demos/python_demos/llm_text_generation/README.md

```diff
@@ -25,18 +32,67 @@ It will create an image called `openvino/model_server:py`
 
 ## Download model
 
-We are going to use [red-pajama-3b-chat](https://huggingface.co/togethercomputer/RedPajama-INCITE-Chat-3B-v1) model in this scenario.
 Download the model using `download_model.py` script:
 
 ```bash
 cd demos/python_demos/llm_text_generation
 pip install -r requirements.txt
-python3 download_model.py
+
+python download_model.py --help
+INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
+usage: download_model.py [-h] --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistal-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+
+Script to download LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+options:
+  -h, --help            show this help message and exit
+  --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistal-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+                        Select the LLM model out of supported list
+
+python download_model.py --model ${SELECTED_MODEL}
+
 ```
+The model will appear in `./tiny-llama-1b-chat` directory.
 
-The model will appear in `./model` directory.
+## Quantization - optional
 
-## Deploy OpenVINO Model Server with the Python Calculator
+Quantization can be applied on the original model. It can reduce the model size and memory requirements. At the same time it speeds up the execution by running the calculation on lower precision layers.
+
+```bash
+python quantize_model.py --help
```

Comment:
Can we change this to be `compress_model.py` or something like that? @dtrawins  Since this is not full quantization, we should use the weight compression terminology.